src/acsets/ACSetInterface.jl

```diff
@@ -39,6 +39,22 @@ function has_part end
 
 function has_subpart end
 
+"""
+Get the domain of a morphism in an acset
+
+domain(acs, f) == parts(acs, X)
+
+where X is the dom of the f in the schema
+"""
+function domain end
```

Comment:
I feel like this shouldn't be `dom`, because the `dom` of the `f` is something else. But you are right this shouldn't be `domain`. Maybe `dom_instantiation`? What do you think?

Reply:
`dom_parts`?

Reply:
Done.